client_plugin/drivers/shared/etcdops.go

```diff
@@ -33,8 +35,18 @@ const (
 	etcdScheme               = "http://"
 	etcdClusterStateNew      = "new"
 	etcdClusterStateExisting = "existing"
+	etcdPrefixState          = "SVOLS_stat_"
+	etcdPrefixGRef           = "SVOLS_gref_"
+	etcdPrefixInfo           = "SVOLS_info_"
+	requestTimeout           = 10 * time.Second
```

Comment:
There is another PR that will have a lot of rebasing to do if I do this now. So I will move them in another checkin which is due soon.